src/main/java/nextstep/subway/path/domain/DistanceFare.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.subway.path.domain;
+
+public enum DistanceFare {
+
+    DISTANCE_10_TO_50(10, 5),
+    DISTANCE_OVER_50(50, 8);
+
+
+    private static final int BASIC_FARE = 1250;
+    private static final int EXTRA_FARE = 100;
+
+    private final int minDistance;
+    private final int extraDistanceUnit;
+
+    DistanceFare(int minDistance, int extraDistanceUnit) {
+        this.minDistance = minDistance;
+        this.extraDistanceUnit = extraDistanceUnit;
+    }
+
+    public static int calculateDistanceFare(int distance) {
+        int fare = BASIC_FARE;
+
+        if (distance > DISTANCE_OVER_50.minDistance) {
```

Comment:
```suggestion
        if(DISTANCE_OVER_50.greaterThan(distance) {
```

직접비교가 아닌 메시지를 전달해보는식도 괜찮겟네요!

Reply:
넵넵